### PR TITLE
Siva

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**

### CODE REVIEW

File: docs/_sidebar.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  - [Public officer](onboard-public-officers-using-non-se-machines)
-    - [WOG Security verification](manage-security-verification-for-wog-account)
-      - [Reset security verification](reset-security-verification-for-wog-account)
-  - [Vendor](onboard-vendors-to-techpass)
-- [Log in to TechPass Portal](log-into-techpass-portal)
-- [Edit Profile](edit-profile)
-- [Access SGTS services](access-sgts-services-using-techpass)
-- [Reset TechPass MFA](reset-techpass-mfa-for-new-device)
+- Getting started
+  - [Public officer onboarding](onboard-public-officers-using-non-se-machines)
```
Comment (yilunyou) [inline docs/_sidebar.md:3]: u might want to check if this menu item has too many chars and can be displayed in the navigation side bar.
if too long then can consider:
Public officer onboard
Vendor onboard

File: docs/access-sgts-services-using-techpass.md [hunk 2]
```diff
@@ -26,13 +28,13 @@ This article guides you on how to use TechPass to access SGTS resources.
...
 
-## To log in with TechPass as public officer
+## Log in as public officer
 
 1. Go to the [Developer portal](https://docs.developer.tech.gov.sg/).
 <kbd>![log-in-with-techpass](assets/images/access-sgts-services-using-techpass/first.png)</kbd>
 2. Sign in to your TechPass account or choose your TechPass account.
 <kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
-3. If you are public officer accessing this SGTS service from your GMD, you will be prompted to authenticate your WOG account. Enter the verification code displayed for your SG Govt M365 on your Authenticator app.
+3. If you are accessing this SGTS service from your GMD, you will be prompted to authenticate your WOG account. Enter the verification code displayed for your SG Govt M365 on your Authenticator app.
```
Comment (yilunyou) [inline docs/access-sgts-services-using-techpass.md:37]: Suggestion:
"... Enter the verification code displayed for your SG Govt M365 profile on your Authenticator app. "

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 3]
```diff
@@ -1,17 +1,16 @@
-# Onboarding public officer using non-SE GSIB device
+# Onboard public officers
 
-This article guides public officers to sign up for TechPass account and to onboard to it using their non-SE GSIB machine.
-
-While public officers create their TechPass using non-SE GSIB machine, they can use their TechPass account to access Singapore Tech Stack(SGTS) products and services from GSIB and GMD machines.
+This article guides public officers to sign up for TechPass account and to onboard to it using their non-SE GSIB machine. They can use their TechPass account to access Singapore Tech Stack(SGTS) products and services from GSIB and GMD machines.
```
Comment (yilunyou) [inline docs/onboard-public-officers-using-non-se-machines.md:3]: The full name of SGTS is Singapore Government Tech Stack
https://www.tech.gov.sg/products-and-services/singapore-government-tech-stack/

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 4]
```diff
@@ -50,16 +51,46 @@ Following are the steps involved in your TechPass onboarding journey.
...
-
 </details>
 
+## Step 3. Set up security verification for your WOG account
+
 <details>
-  <summary>c. Onboard to TechPass</summary><br>
+  <summary>How to set up security verification for WOG account?</summary>
+
+  Public officers need to set up security verification(multi-factor authentication) for their Whole-of-Government(WOG) account to access Singapore Tech Stack (SGTS) services and tools from their GMD device.
```
Comment (yilunyou) [inline docs/onboard-public-officers-using-non-se-machines.md:61]: Singapore Government* ....

File: docs/_sidebar.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply docs/_sidebar.md:3]: Done

File: docs/access-sgts-services-using-techpass.md (hunk 2 above)
Comment (SenthilKumarSivakami) [reply docs/access-sgts-services-using-techpass.md:37]: Done, thank you!

File: docs/onboard-public-officers-using-non-se-machines.md (hunk 3 above)
Comment (SenthilKumarSivakami) [reply docs/onboard-public-officers-using-non-se-machines.md:3]: Done, thank you!

Comment (SenthilKumarSivakami) [reply docs/onboard-public-officers-using-non-se-machines.md:3]: On the home.md page it is mentioned as Technology Stack, so I too will use the same for consistency.

Comment (SenthilKumarSivakami) [reply docs/onboard-public-officers-using-non-se-machines.md:3]: Singapore Government Technology Stack (SGTS)

**Pull request merged**